### PR TITLE
Moved entries and webhook docs

### PR DESCRIPTION
**Pull request by alexbaldwin**

### CODE REVIEW

File: source/includes/_entries.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+signed document PDF.
+
+## Location visitors
+
+```
+GET https://app.signwithenvoy.com/api/entries.json?api_key=API_KEY
+```
+
+```json
+{
```
Comment (kamal) [inline source/includes/_entries.md]: This is missing the top-level array. Also, this only supports pulling down the entries for the master location. This endpoint also supports date ranges.

File: source/includes/_webhooks.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Webhooks
+
+Create custom workflows that get triggered when a visitor signs in on the iPad. You can see the details on your [Settings > Integrations](https://dashboard.signwithenvoy.com/settings/integrations) and add a webhook for your custom integration.
+
+A webhook makes Envoy servers perform an HTTP request to the specified URL whenever a new visitor signs into your office. The request includes a `POST` with the following parameters:
+
+
+| Name      | Description |
+| --------- | ----------- |
+| `entry` | **String** _(optional)_ Details about the visitor. |
```
Comment (kamal) [inline source/includes/_webhooks.md]: this should expand the `entry` json

Comment (kamal) [inline source/includes/_webhooks.md]: `entry` is not optional

File: source/includes/_webhooks.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+| `status` | **String** _(optional)_ Either `sign_in` or `sign_out`. |
+| `timestamp` | **DateTime** _(optional)_ Returned in UTC as Unix time. |
+| `token` | **String** _(optional)_ Random string. |
+
+A signature of the parameters (to ensure the request is actually coming from us). The signature is calculated as such: `OpenSSL::HMAC.hexdigest (OpenSSL::Digest::Digest.new ('sha256'), api_key, '%s%s' % [timestamp, token])`
+
+A great way to test this and experiment is to use [RequestBin](http://requestb.in/) (but **only** for testing!)
+
+Note that we retry accessing your servers 5 times over the span of 10 minutes, after which the request will be dropped if unsuccessful.
```
Comment (kamal) [inline source/includes/_webhooks.md]: Also make a note if the webhook URL is SSL, it must be a valid SSL certificate.

**Comment by alexbaldwin** [comment]
:+1:

File: source/includes/_webhooks.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Webhooks
+
+```json
+{
+   "id":1386119,
```
Comment (kamal) [inline source/includes/_webhooks.md]: eh but this is not how the payload looks like

**Pull request merged**